src/main/java/MainWindow.java

```diff
@@ -0,0 +1,51 @@
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+import duke.core.*;
```

Comment:
Should the import statements be more explicitly stated instead of using *?

Reply:
That's true